README.md

```diff
@@ -75,17 +72,35 @@ from load_confounds import Params6
+
+
+## Recreating strategies from Ciric et al. 2017
+
+`load_confounds` can recreate the following strategies. The following table highlights the relevant options:
+
+| Strategy        | `high_pass` | `motion` | `wm_csf` | `global` | `compcor` | `ica_aroma` | `scrub` |
+| --------------- |:-----------:|:--------:|:--------:|:--------:|:---------:|:-----------:|:-------:|
+| `Params2`       | x           |          | `basic`  |          |           |             |         |
+| `Params6`       | x           | `basic`  |          |          |           |             |         |
+| `Params9`       | x           | `basic`  | `basic`  | `basic`  |           |             |         |
+| `Params9Scrub`  | x           | `basic`  | `basic`  |          |           |             | `full`  |
+| `Params24`      | x           | `full`   |          |          |           |             |         |
+| `Params36`      | x           | `full`   | `full`   | `full`   |           |             |         |
+| `Params36Scrub` | x           | `full`   | `full`   |          |           |             | `full`  |
+| `AnatCompCor`   | x           | `full`   |          |          | `anat`    |             |         |
+| `TempCompCor`   | x           |          |          |          | `temp`    |             |         |
+| `ICAAROMA`      | x           |          | `basic`  |          |           | `full`      |         |
+| `AROMAGSR`      | x           |          | `basic`  | `basic`  |           | `full`      |         |
+| `AggrICAAROMA`  | x           |          | `basic`  | `basic`  |           | `basic`     |         |
 
-## A note about ICA-AROMA denoising
-ICA-AROMA related strategies (`ICAAROMA`, `AROMAGSR`, `AggrICAAROMA`) are only applicable to fMRIprep output generated with `--use-aroma`. There are two approaches to apply the noise independent components. Ciric et al. (2017) benchmarked the non-aggressive apporach (`ICAAROMA`, `AROMAGSR`) involving two steps. Firstly it estimates signals with a linear regression including all independent components, and then performs partial regression on variance associated with noise independent components. fMRIprep perfroms the first step and produces files with suffix `desc-smoothAROMAnonaggr_bold`. Other noise regressors, such as WM/CSF, were removed in the second step, implemented by `load_confounds`. The aggressive approach (`AggrICAAROMA`) was described in Pruim et al. (2015) and achieve denoising in one step by `load_confound`. Noise independent components along with other source of noise are included in confound regressors. This approach must be applied to the regular minimally processed fMRIprep output suffixed `desc-prepro_bold`. The name "aggressive" reflects that this approach doesn't consider the potential good signals regressed out by the noise independent compoenents. Please refer to relevant stategies in `load_confounds` for more details.
 
```

Comment:
codacy is complaining of two successive blank lines before `## Funding` (it wants just one)